core/src/main/java/org/radargun/stats/MultiStatistics.java

```diff
@@ -0,0 +1,118 @@
+package org.radargun.stats;
+
+import java.util.Arrays;
+import java.util.Set;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import org.radargun.Operation;
+
+/**
+ * @author Radim Vansa &lt;rvansa@redhat.com&gt;
+ */
+public abstract class MultiStatistics implements Statistics {
```

Comment:
Do you expect more subclasses in future? Because right now this has only 1 subclass.